docker-compose/run_selenium_tests_local_keybase.sh

```diff
@@ -0,0 +1,213 @@
+    echo
+    echo "    mslsx (default) = use MSLSX for identity service."
+    echo
+    exit 1
+  else
+    if [ $1 == "slsx" ]
+    then
+      USE_MSLSX=false
+      DJANGO_MEDICARE_SLSX_REDIRECT_URI="http://bb2slsx:8000/mymedicare/sls-callback"
+      DJANGO_MEDICARE_SLSX_LOGIN_URI="https://test.medicare.gov/sso/authorize?client_id=bb2api"
+      DJANGO_SLSX_HEALTH_CHECK_ENDPOINT="https://test.accounts.cms.gov/health"
+      DJANGO_SLSX_TOKEN_ENDPOINT="https://test.medicare.gov/sso/session"
+      DJANGO_SLSX_SIGNOUT_ENDPOINT="https://test.medicare.gov/sso/signout"
+      DJANGO_SLSX_USERINFO_ENDPOINT="https://test.accounts.cms.gov/v1/users"
+    fi
+  fi
+fi
+
+# Set KeyBase ENV path based on your type of system
+SYSTEM=$(uname -s)
+
+echo_msg " - Setting Keybase location based on SYSTEM type: ${SYSTEM}"
+echo_msg
+
+if [[ ${SYSTEM} == "Linux" ]]
+then
+  keybase_env_path="/keybase"
+elif [[ ${SYSTEM} == "Darwin" ]]
+then
+  keybase_env_path="/Volumes/keybase"
+else
+  # support cygwin
+  keybase_env_path="/cygdrive/k"
+  CERTSTORE_TEMPORARY_MOUNT_PATH="./docker-compose/certstore"
+  DJANGO_FHIR_CERTSTORE="/code/docker-compose/certstore"
+fi
+
+# Keybase ENV file
+keybase_env="${keybase_env_path}/${KEYBASE_ENV_FILE}"
+
+echo_msg " - Sourcing ENV secrets from: ${keybase_env}"
+echo_msg
+
+# Check that ENV file exists in correct location
+if [ ! -f "${keybase_env}" ]
+then
+echo_msg
+echo_msg "ERROR: The ENV secrets could NOT be found at: ${keybase_env}"
+echo_msg
+exit 1
+fi
+
+# Source ENVs
+source "${keybase_env}"
+
+# Check ENV vars have been sourced
+if [ -z "${DJANGO_USER_ID_SALT}" ]
+then
+echo_msg "ERROR: The DJANGO_USER_ID_SALT variable was not sourced!"
+exit 1
+fi
+if [ -z "${DJANGO_USER_ID_ITERATIONS}" ]
+then
+echo_msg "ERROR: The DJANGO_USER_ID_ITERATIONS variable was not sourced!"
+exit 1
+fi
+
+# Check temp certstore dir and create if not existing
+if [ -d "${CERTSTORE_TEMPORARY_MOUNT_PATH}" ]
+then
+echo_msg
+echo_msg "  - OK: The temporary certstore mount path is found at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+else
+mkdir ${CERTSTORE_TEMPORARY_MOUNT_PATH}
+echo_msg
+echo_msg "  - OK: Created the temporary certstore mount path at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+fi
+
+# Keybase cert files
+keybase_certfiles="${keybase_env_path}/${KEYBASE_CERTFILES_SUBPATH}"
+keybase_cert_file="${keybase_certfiles}/${CERT_FILENAME}"
+keybase_key_file="${keybase_certfiles}/${KEY_FILENAME}"
+
+# Check that certfiles in keybase dir exist
+if [ -f "${keybase_cert_file}" ]
+then
+    echo_msg
+    echo_msg "  - OK: The cert file was found at: ${keybase_cert_file}"
+else
+    echo_msg
+    echo_msg "ERROR: The cert file could NOT be found at: ${keybase_cert_file}"
+    exit 1
+fi
+
+if [ -f ${keybase_key_file} ]
+then
+    echo_msg
+    echo_msg "  - OK: The key file was found at: ${keybase_key_file}"
+else
+    echo_msg
+    echo_msg "ERROR: The key file could NOT be found at: ${keybase_key_file}"
+    exit 1
+fi
```

Comment:
Can we make the indentation consistent across all if else statements?

Reply:
applied with shell formatter e.g. beautysh